platform/podjs/src/manifest-parser/src/manifest.ts

```diff
@@ -67,13 +68,20 @@ const featureDecoder = Decode.type({
     translations: Decode.UnknownRecord,
 });
 
-export async function readManifest(packageManifest: Record<string, unknown>): Promise<Manifest> {
+export async function readManifest(
+    packageManifest: Record<string, unknown>
+): Promise<Manifest> {
     // There is no 'version' property in real feature manifests at this time, but to make this code
     // happy, we're simply adding one if it's missing.
     const manifestWithVersion = { ...packageManifest };
-    if (!("version" in manifestWithVersion)) manifestWithVersion.version = "1.0.0";
+    if (!("version" in manifestWithVersion))
+        manifestWithVersion.version = "1.0.0";
```

Comment:
You can probably use this to eliminate this code, and all the "version" stuff, that was just introduced to avoid CI errors.

Reply:
Yeah, we could get rid of the version stuff actually - it's nice how much thought went into it, but we do not actually need or use it right now :D

Reply:
I guess with this we no need the `no-version.json` test-case anymore, right?

Reply:
No, we don't...